IN-global_conf.json

```diff
@@ -1,78 +1,79 @@
 {
 	"SX1301_conf": {
 		"lorawan_public": true,
-		"clksrc": 1,
+		"clksrc": 0,
```

Comment:
Please leave this to `1`. For the MultiTech, the packet forwarder or gateway agent should run a post processor setting this to `0`